must-gather/collection-scripts/gather_logstore_resources

```diff
@@ -111,7 +116,7 @@ done
 anypod=""
 for comp in "elasticsearch"
 do
-    echo "-- Getting Elasticsearch cluster info from logging-${comp} pod"
+    echo "-- Getting Elasticsearch  testing cluster info from logging-${comp} pod"
```

Comment:
Why would we alter the the message here?

Reply:
@jcantrill  I have removed  the testing word it was accidentally added  for testing changes  now i have removed it 

>> Modify the description to fully expand what this change is accomplishing
For this, i have added details in the PR request as well as in the new commit added a short description let me know if i am missing something